server/container/guice/memory-guice/src/main/java/org/apache/james/modules/data/MemoryDataModule.java

```diff
@@ -61,6 +63,9 @@ protected void configure() {
         bind(MemoryAccessTokenRepository.class).in(Scopes.SINGLETON);
         bind(AccessTokenRepository.class).to(MemoryAccessTokenRepository.class);
 
+        bind(MemoryVacationRepository.class).in(Scopes.SINGLETON);
```

Comment:
we have a mix of two things here : classes for jmap and others for `data`. It should be two modules.